ui/src/views/repository-data/components/sync-types-table/columns.tsx

```diff
@@ -10,13 +10,14 @@ import {
   RepositorySyncStatus,
   RepositoryTableRowOptions,
 } from './components'
+import { TimeAgoField } from '../../../../components/Fields/time-ago-field'
```

Comment:
this is just a question - is it possible to use absolute paths in imports? From a fixed starting point like `src/`? If so, what do you think about standardizing on that for clarity?

Reply:
I fixed this specific import to be absolute. As far standardize it, Do you mean to set up `src` as a root in `baseUrl` like [this](https://medium.com/geekculture/making-life-easier-with-absolute-imports-react-in-javascript-and-typescript-bbdab8a8a3a1)?, If so, we need do it in other issue cuz it will need to be refactor a lot of imports through all project.

Reply:
for standardizing I just mean using absolute imports over relative imports everywhere for consistency - no need to update the `baseUrl` IMO